src/lib/histogram.ts

```diff
@@ -714,7 +714,7 @@ export default function makeHistogram(): Histogram {
             .join('text')
             .attr('class', 'histogram-title')
             .attr('x', width / 2 )
-            .attr('y', 4)
+            .attr('y', -margins.top / 2 + 5)
```

Comment:
Yup, I like this much better. Perhaps we could even do `            .attr('y', -margins.top / 2 + (margins.top / 4))`? That way, the title is always 3/4 of the way into the margin and it isn't hardcoded to 5px even if the margin is like 200px (not sure why anyone would do that, but at least we'd handle it gracefully).

Reply:
Yup, this is better.